SpatialDE/_internal/score_test.py

```diff
@@ -273,3 +273,44 @@ def _grad_negative_negbinom_loglik(params, counts, sizefactors):
             + (counts - mus) / one_alpha_mu
         )  # d/d_alpha
         return -tf.convert_to_tensor((grad0, grad1))
+
+
+class NormalScoreTest(ScoreTest):
+    @dataclass
+    class NullModel(ScoreTest.NullModel):
+        mu: tf.Tensor
+        sigma: tf.Tensor
+
+    def _fit_null(self, y: tf.Tensor) -> NullModel:
+        return self.NullModel(tf.reduce_mean(y), tf.reduce_std(y))
```

Comment:
One minor thing: Perhaps it would make more sense to calculate the variance here and remove the square in `_do_test`?

Reply:
OK, made changes accordingly